mongodbatlas/fw_resource_mongodbatlas_alert_configuration_migration_test.go

```diff
@@ -0,0 +1,92 @@
+package mongodbatlas
```

Comment:
Can we add test cases for each collection in the resource (empty/null if optional, multiple elements wherever possible)
I have had to add logic for null collections, hence we should make sure

Reply:
Added additional migration tests to verify all scenarios:
- empty/multiple matchers
- empty/defined metric threshold config
- empty/defined threshold config